### DISCUSSION TITLE
Welcome to TiddlyWiki5 Developer Discussions

### DISCUSSION THREAD

**Discussion by Jermolene** [TiddlyWiki/TiddlyWiki5#5247] [General]
## Welcome!

Welcome to the TiddlyWiki5 developer discussion group. This is the place for talking about TiddlyWiki core internals and development,  and plugin development issues. For general assistance with TiddlyWiki5 features, macros, formatting, documentation, etc., please see the [TiddlyWiki group](https://groups.google.com/group/TiddlyWiki).

When posting a question, please use a short "[keyword]" in the subject line to indicate the general area of concern.  Include your OS, Browser, Tiddlywiki version and other relevant details in your message so that the community can help you better.
Visit tiddlywiki.com to use, learn, read about & download TiddlyWiki.
Discussions about TiddlyWiki Classic have moved to the TiddlyWikiClassic group
An enhanced group search facility is available via mail-archive.com

Please use this thread for discussion about our adoption of GitHub Discussions.

**Comment by msameen99** [comment]
Nice move Jeremy,  it's more intuitive to have the code and related discussion in the same place

**Comment by kookma** [comment]
@Jermolene - please pin your above post 

There is also categories, when someone posts he/she has to select a category, so you can add more categories if you think its better for directing discussions.

**Comment by kookma** [comment]
@Jermolene 
 The current categories on Tiddlywiki discussion panel are

- general
- ideas
- Q&A
- Show and tell

You may add some specific categories if you think it is better to have different areas for discussions. 

I am not sure if who make a question can select the best answer or it is the admin can do this

**Comment by saqimtiaz** [comment]
@Jermolene this will be helpful. I've created issues/PRs in the last weeks just to have a discussion as this is where most developers are active.

Hopefully we can also convert some current issues and PRs into discussions. At the moment it takes a lot of time to find issues that are actionable, i.e. where this consensus on what to do and only the implementation is lacking.

**Reply by kookma** [reply]
@saqimtiaz - I think it should be recommended in Jeremy welcome message, always start discussion, and when every aspects were clear, and the discussion is actionable, then author coverts his/her discussion to suitable ticket (PR or Issue).

**Comment by DeRuiterIndustrieleAutomatisering** [comment]
Code and related discussion in the same place is an advantage. But is there a RSS feed? The big advantage for me of the Google Groups was easy reading with a RSS reader.

**Reply by Jermolene** [reply]
Sadly, GitHub doesn't support RSS or Atom.

**Reply by DeRuiterIndustrieleAutomatisering** [reply]
I will miss the RSS/Atom feeds and the topic digests by email. I hope github will add these in the near future. For now I have enabled notifications for every new post

**Comment by saqimtiaz** [comment]
@Jermolene can we change the welcome message to make it clear these are developer discussions rather than end-user? 

We are starting to get posts here about end-user discussions that are better suited for the google group.

**Reply by saqimtiaz** [reply]
I am referring to this welcome message:

![image](https://user-images.githubusercontent.com/68092/102582072-1e5bba00-4102-11eb-889c-af1ed5c1ac85.png)

**Reply by Jermolene** [reply]
Thanks @saqimtiaz I've added the word "Developer". Annoyingly one doesn't seem to be able to change the banner image.

**Comment by dixonge** [comment]
Is there any way to pin this Welcome message to the top?

**Reply by Jermolene** [reply]
This conversation is actually already pinned to the top, GitHub just has a slightly strange way of implementing it: it appears as the banner at the top of the discussion page (see the illustration above).

**Reply by dixonge** [reply]
yeah, I had NO idea that was a link! Very odd UI/UX decision there. Oh well, I tried LOL Thanks for the info!
